packages/react-scripts/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@skyscanner/backpack-react-scripts",
-  "version": "10.0.1",
+  "version": "11.0.0",
```

```suggestion
  "version": "10.0.1",
```
The version can be changed when publishing through `npm version major`.